keyboards/skeletonkbd/frost68/config.h

```diff
@@ -0,0 +1,90 @@
+// Copyright 2023 skeletonkbd (@skeletonkbd)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+/*
+ * Feature disable options
+ *  These options are also useful to firmware size reduction.
+ */
+
+/* disable debug print */
+// #define NO_DEBUG
+
+/* disable print */
+// #define NO_PRINT
+
+/* disable action features */
+// #define NO_ACTION_LAYER
+// #define NO_ACTION_TAPPING
+// #define NO_ACTION_ONESHOT
+
+#define DEBOUNCE 50
```

Comment:
Do you really need this big of a debounce? Anyway, this config should be moved to the `info.json` file

Reply:
Now I don't use it. So I removed this line.
f7c05da5c01e8ffc697e034d119423af475bc826